selfservice/hook/verification_test.go

```diff
@@ -115,7 +115,11 @@ func TestVerifier(t *testing.T) {
 
 						messages, err := reg.CourierPersister().NextMessages(context.Background(), 12)
 						require.NoError(t, err)
-						require.Len(t, messages, 2)
+						if enabled {
+							require.Len(t, messages, 2)
+						} else {
+							require.Len(t, messages, 1)
```

Comment:
the new behavior sends only one email, for the first unverified email it finds. it doesn't make sense in my view to send a code to all emails, because it's unclear to the user which code they have to enter in the next screen.